src/main/scala/edu/berkeley/ce/rockslicing/NumericUtils.scala

```diff
@@ -25,4 +26,18 @@ object NumericUtils {
    */
   def applyTolerance(values: Seq[Double]): Seq[Double] =
     values map applyTolerance
+
+  /**
+    * Rounds to the specified number of decimal places. If not specified, default
+    * is 6 decimal places
+    * @param d A Double value
+    * @param decimals Number of decimal places to round to
+    * @return Value with 6 decimal places
```

Comment:
Now that the number of decimal places can be specified, you'll want to update this documentation.